sormas-backend/src/main/resources/sql/sormas_schema.sql

```diff
@@ -9798,4 +9798,27 @@ ALTER TABLE weeklyreportentry_history ALTER COLUMN creationdate TYPE timestamp(3
 
 INSERT INTO schema_version (version_number, comment) VALUES (436, 'Set timestamp precision to milliseconds #7303');
 
+-- 2022-01-25 Add configurations for each entity to trigger automated deletion #7008
+CREATE TABLE deletionconfiguration(
+                                     id bigint not null,
+                                     uuid varchar(36) not null unique,
+                                     changedate timestamp not null,
+                                     creationdate timestamp not null,
+                                     entityType varchar(255),
+                                     deletionReference varchar(255),
```

Comment:
Inconsistent not null constraint definitions compared to the entity definition for both deletionReference and deletionPeriod

Reply:
since the deletion reference and deletion period could be undefined "nullable = false" was removed from the respective fields in the entity class DeletionConfiguration